### PR TITLE
Properly tags Plural-able strings for translation, and removes unnecessary i18n

### LINKED ISSUE #9891: Delete warning showing some plural ploblems
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
**Note:** this may or may not exist in different places/pages I found this trying to delete resource that any other resource is using, the delete warning shows like:

```
Warning alert:
{numItemsToDelete, plural, one {This execution environment is currently being used by other resources. Are you sure you want to delete it?} other {Deleting these execution environemnts could impact other resources that rely on them. Are you sure you want to delete anyway?}}
```

##### ENVIRONMENT
* Devel

##### STEPS TO REPRODUCE

Create EEs used by any org, then try to delete the EE

##### EXPECTED RESULTS

image

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
This resolves #9891.

It also begings the process of remove `i18n._(t string)` in places in favor of [this syntax](https://github.com/ansible/awx/compare/devel...AlexSCorey:9891-Plural?expand=1#diff-8233f6a68481569a3d0ffa0df813b8ecfb39348b402791c37177906e259dbf94R168)
##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

##### AWX VERSION

```

```

##### ADDITIONAL INFORMATION
Screen Shot 2021-04-13 at 11 59 43 AM

Screen Shot 2021-04-13 at 11 59 55 AM

### CODE REVIEW

**Comment by unlikelyzero** [comment]
@AlexSCorey can you run the visual tests and functional tests?

File: awx/ui_next/src/components/PaginatedDataList/ToolbarDeleteButton.jsx [hunk 1]
```diff
@@ -145,11 +145,16 @@ function ToolbarDeleteButton({
...
-            : i18n._(
-                t`You do not have permission to delete ${pluralizedItemName}: ${itemsUnableToDelete}`
-              )}
+          {errorMessage ? (
+            <>
+              <span>{errorMessage}</span>
+              <span>{`: ${itemsUnableToDelete}`}</span>
+            </>
+          ) : (
+            i18n._(
```
Comment (keithjgrant) [inline awx/ui_next/src/components/PaginatedDataList/ToolbarDeleteButton.jsx]: can this just be t`` now?

**Review by keithjgrant — APPROVED** [review]
🎉

File: awx/ui_next/src/components/PaginatedDataList/ToolbarDeleteButton.jsx (hunk 1 above)
Comment (nixocio) [reply awx/ui_next/src/components/PaginatedDataList/ToolbarDeleteButton.jsx]: If usage of `t` is not going to completely remove `i18n._(` we will have 2 standards for translation syntax. It there any benefit in using one over the other?

File: awx/ui_next/src/screens/ExecutionEnvironment/ExecutionEnvironmentList/ExecutionEnvironmentList.jsx [hunk 2]
```diff
@@ -182,12 +181,15 @@ function ExecutionEnvironmentList({ i18n }) {
...
                     deleteDetailsRequests={deleteDetailsRequests}
-                    deleteMessage={i18n._(
-                      '{numItemsToDelete, plural, one {This execution environment is currently being used by other resources. Are you sure you want to delete it?} other {Deleting these execution environemnts could impact other resources that rely on them. Are you sure you want to delete anyway?}}',
-                      { numItemsToDelete: selected.length }
-                    )}
+                    deleteMessage={
+                      <Plural
+                        value={selected.length}
+                        one="This execution environment is currently being used by other resources. Are you sure you want to delete it?"
+                        other="Deleting these execution environemnts could impact other resources that rely on them. Are you sure you want to delete anyway?"
```
Comment (nixocio) [inline awx/ui_next/src/screens/ExecutionEnvironment/ExecutionEnvironmentList/ExecutionEnvironmentList.jsx]:
```suggestion
                        other="Deleting these execution environments could impact other resources that rely on them. Are you sure you want to delete anyway?"
```

File: awx/ui_next/src/components/PaginatedDataList/ToolbarDeleteButton.jsx (hunk 1 above)
Comment (AlexSCorey) [reply awx/ui_next/src/components/PaginatedDataList/ToolbarDeleteButton.jsx]: In most cases we will be able to remove `withI18n()` HOC and also i18n that is being injected into the components in favor of simply using `t`

**Comment by mabashian** [comment]
@AlexSCorey looks like this one is gonna need a rebase

**Review by tiagodread — CHANGES_REQUESTED** [review]
@AlexSCorey I'm facing the same problem in different places (dev or production build) like:

image

**Review by tiagodread — APPROVED** [review]
I could not reproduce the problem anymore in a manual exploration testing, the e2e tests are 🟢

**Comment by tiagodread** [comment]
regate

**Pull request merged**